config.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from functools import lru_cache
+from typing import Union, Optional
+from pydantic_settings import BaseSettings, SettingsConfigDict
+from pydantic import SecretStr, PostgresDsn, DirectoryPath, Field, validator
+
+logger = logging.getLogger(__name__)
+
+class Settings(BaseSettings):
+    model_config = SettingsConfigDict(env_file="/home/abdullah/Documents/hdd/projects/ansari/ansari-backend/.env", env_file_encoding="utf-8", case_sensitive=True)
+    
+    DATABASE_URL: PostgresDsn = Field(default="postgresql://mwk@localhost:5432/mwk")
```

Comment:
Should probably get rid of this too :-).

Reply:
Done